test/NamingStrategy/MapNamingStrategyTest.php

```diff
@@ -31,14 +48,14 @@ public function invalidMapValues(): iterable
     /** @psalm-return iterable<string, array{0: mixed, 1: null|string}> */
     public function invalidKeyValues(): iterable
     {
-        // phpcs:disable WebimpressCodingStandard.NamingConventions.ValidVariableName.NotCamelCaps
-        $isPHP81 = PHP_VERSION_ID >= 80100;
         yield 'null'       => [null, null];
         yield 'true'       => [true, null];
         yield 'false'      => [false, null];
         yield 'zero-float' => [0.0, null];
-        yield 'float'      => [1.1, $isPHP81 ? 'Implicit conversion' : null];
-        // phpcs:enable
+        if (PHP_VERSION_ID >= 80100) {
+            error_reporting(E_ALL & ~E_DEPRECATED);
```

Comment:
We should really not raise runtime deprecation warnings from this library 🤔

Reply:
> We should really not raise runtime deprecation warnings from this library

Sorry, I couldn't understand all.
This test's `error_reporting(E_ALL & ~E_DEPRECATED)` aims to suppress when making test data array.

With 8429755, I changed only when creating array in order to reduce impact.

Reply:
> aims to suppress when making test data array.

Is the error only in the data provider? If so, we fix the data provider (rather than suppressing the deprecation)

In practice, if the type is incompatible, we should either throw an exception, or ignore the value, but never produce a warning of any kind.